src/test/java/uk/gov/justice/hmpps/prison/service/BookingServiceTest.java

```diff
@@ -804,99 +853,6 @@ public void getBookingVisitsWithVisitor_noVisitors() {
                         .build());
     }
 
-    @Test
-    public void getBookingVisitsWithVisitor_visitorNoPerson() {
```

Comment:
This test doesn't seem to make any sense, it is trying to define a scenario where a contact does not relate to a person which is impossible. So deleted test